prac_07/guitars.py

```diff
@@ -0,0 +1,24 @@
+from prac_07.guitar_storage import Guitar
+
+print("My guitars!")
+name = "j"
```

Comment:
What is this? Where have you seen us do this?
Don't just make stuff up... it's not good.
Here's how (we've taught you): 
https://github.com/CP1404/Starter/wiki/Programming-Patterns#while-loops